cli/src/config.ts

```diff
@@ -154,9 +154,12 @@ async function loadAndroidConfig(
   const name = 'android';
   const platformDir = extConfig.android?.path ?? 'android';
   const platformDirAbs = resolve(rootDir, platformDir);
-  const webDir = 'app/src/main/assets/public';
-  const resDir = 'app/src/main/res';
-  const buildOutputDir = 'app/build/outputs/apk/debug';
+  const appDir = 'app';
+  const srcDir = `${appDir}/src/main`;
```

Comment:
Why is it called `srcDir ` but points to `main` dir instead of to `src`?
We should probably have both, with appropriate names.

Reply:
I can do that, I just saw that the `main` source set appears to be where everything is (besides tests).

Reply:
in regular apps yes, but there can also be `debug`, `release` and other folders inside `src`, so might need to have a src path and a main path